### PR TITLE
Parse Party as String

### PR DESCRIPTION
**Pull request by sgelbart**

### CODE REVIEW

File: app/services/importers/party.rb [hunk 1]
```diff
@@ -18,7 +18,13 @@ def self.perform(parties_json, court_case, logs)
 
     def perform
       parties_json.each do |party_data|
-        save_parties(party_data)
+        if party_data[:link].nil?
+
+          save_parties_text(party_data)
+        else
+          #  binding.pry
```
Comment (sgelbart) [inline app/services/importers/party.rb:25]: remove

File: app/services/importers/party.rb [hunk 2]
```diff
@@ -18,7 +18,13 @@ def self.perform(parties_json, court_case, logs)
...
+          save_parties(party_data)
+        end
```
Comment (sgelbart) [inline app/services/importers/party.rb:27]:
```suggestion
        if party_data[:link].present?
            save_parties(party_data)
        else
            save_parties_text(party_data)
        end
```

File: app/models/party.rb [hunk 3]
```diff
@@ -9,8 +9,7 @@ class Party < ApplicationRecord
   has_many :addresses, class_name: 'PartyAddress', dependent: :destroy
   has_many :aliases, class_name: 'PartyAlias', dependent: :destroy
   has_one :party_html, dependent: :destroy
-
-  validates :oscn_id, uniqueness: { case_sensitive: true }
+
```
Comment (sgelbart) [inline app/models/party.rb:12]: Remove whitepace to fix rubocop?
```suggestion
```

File: app/services/importers/party.rb [hunk 4]
```diff
@@ -35,6 +39,10 @@ def save_parties(party_data)
       end
     end
 
+    def save_parties_text(party_data)
+      create_and_save_party_to_case_text(party_data)
```
Comment (sgelbart) [inline app/services/importers/party.rb:43]: Does this handle multiple parties?

File: spec/services/importers/court_case_spec.rb [hunk 5]
```diff
@@ -13,13 +15,34 @@
...
     end
+
+    it 'saves party with text only for a case' do
+      # Create a case with html from fixture
+      county = create(:county, name: 'oklahoma')
+
+      create(:judge, name: 'Collins, April')
+      court_case = create(:court_case, county_id: county.id, case_number: 'CS-2022-1020')
+      html = File.read('spec/fixtures/oscn_id_less.html')
+      create(:case_html, court_case: court_case, html: html)
```
Comment (sgelbart) [inline spec/services/importers/court_case_spec.rb:33]: These tests look good but the standard in rspec is to create objects in the let! blocks (I'll show you how if you want)
This is fine for now

**Review by AspiringDev51 — APPROVED** [review]

**Pull request merged**